src/ConformalTracking.cc

```diff
@@ -455,7 +461,8 @@ void ConformalTracking::processEvent(LCEvent* evt) {
       std::sort(trackHits.begin(), trackHits.end(), sort_by_radiusKD);
 
       // Make a track
-      auto mcTrack = std::unique_ptr<KDTrack>(new KDTrack());
+      auto pars    = _stepParameters[0];
```

Comment:
@andresailer not sure what to set here (new mc track in debug loop). I have used the same solution adopted in the function checkReconstructionFailure()